WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -858,9 +858,6 @@ extension WordPressAppDelegate {
     private func customizeAppearanceForTextElements() {
         let maximumPointSize = WPStyleGuide.maxFontSize
 
-        UINavigationBar.appearance().titleTextAttributes = [NSAttributedString.Key.foregroundColor: UIColor.white,
```

Comment:
Hmmm. I'm a bit worried about removing that from the `AppDelegate`. We might end affecting some other `UINavigationBar`. Do you think we can keep that here and customize the comments nav bar?

Reply:
Unfortunately I was not able to get that to work on older iOS versions (iOS 12 and below). No matter what I did the title color would always be white, and removing this line is the only thing that worked. 

I went through a number of flows, and didn't see any changes anywhere. 

Now that I think about it I should have made this its own separate PR so we can more easily determine if it's something that we want to do.  Thoughts?

Reply:
After a discussion with @frosty about this issue on Slack he came up with a fix that does not involve removing this line.

Reply:
After poking around some more I realized that when configuring the appearance for the `LightNavigationController` we weren't setting the `titleTextAttributes` at all, adding that line in fixes this issue.